tests/shared/file-extensions.spec.js

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2016 resin.io
```

Comment:
As I've said before (I think!), IMHO the year should only reflect the date that the file was first added. No massive search&replace needed ;P